financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,167 @@
+= RPA Setup Guide
+
+This guide provides instructions for implementing the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-financial-services/minor/{fins-version}/pages/Use%20case%207%20-%20Unlock%20AMS%20through%20hyperautomation%20for%20insurance%20brokers/[Unlock AMS Through Hyperautomation for Insurance Brokers] use case, which is part of the MuleSoft Accelerator for Financial Services.
+
+The use case leverages MuleSoft's Robotic Process Automation (RPA) solution to retrieve customer data from your Agency Management System (AMS) and automatically load the data into your CRM. To implement this use case, you must set up RPA on the device that runs the automation process and configure the solution to your specific AMS.
+
+== Before You Begin
+
+* Create an Anypoint Platform account xref:rpa-home::automation-userrolesandpermissions-anypointrpapermissions.adoc[and assign it the `RPA Automations Manager` and `RPA Bots Manager` permissions].
+* Download the latest version of MuleSoft RPA Manager, RPA Builder, and RPA Bot.
```

Comment:
RPA Manager is a cloud solution that belongs to the MuleSoft Automation bundle, so this is provided and has not to be installed. You have to download RPA Builder and RPA Bot from the RPA Manager Home Page and then connect them to your RPA Manager instance. 

- https://docs.mulesoft.com/rpa-builder/latest/install-and-configure
- https://docs.mulesoft.com/rpa-bot/latest/installation

Reply:
You have to installl Bot and Builder on the computer from where you can access the AMS.